notebooks/00_quick_start/lightgbm_tinycriteo.ipynb

```diff
@@ -0,0 +1,802 @@
+{
```

Comment:
<p>This notebook will give you a quick example of how to train LightGBM model in recommendation scenario.</p><p><br></p><p>to</p><p><br></p><p>This notebook will give you a quick example of how to train LightGBM model in a recommendation scenario.</p>

 _Reply via <a href='https://app.reviewnb.com/Microsoft/Recommenders/pull/611/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
Fixed.